src/main/java/com/kruize/analysis/AnalysisImpl.java

```diff
@@ -75,8 +76,16 @@ public void calculateCpuLimit(MetricsImpl instance)
             return;
         }
 
+        if (instance.getName().contains("petclinic")) {
```

Comment:
yikes ... is this by mistake, why are we checking for petclinic here ?

Reply:
Oops, yes looks like I left some test code by mistake. Removing it

Reply:
Removed all test code in latest commit